bundler/lib/bundler/cli/gem.rb

```diff
@@ -310,6 +315,7 @@ def hint_text(setting)
     end
 
     def ask_and_set_ci
+      return false if skip?(:ci)
```

Comment:
Would be `return if skip?(:ci)` enough here?

Reply:
It would work too, I return false to be consistent with the method code below that gets executed when `options[:ci]` is nil/empty and returns false if the provided value is not in the regex list:

https://github.com/rubygems/rubygems/blob/c1a7fbdd4abf0379cfe507a5cbf1c790cae4c4bd/bundler/lib/bundler/cli/gem.rb#L325-L329

If `return if skip(:ci)` is the preferred style let me know and I can change it

Reply:
If value not used, I prefer to not provide any to keep it simple. Indeed ci method needs this (I have not left comment on that one). Each method in here is isolated and there is no expectation to share anything IMHO.

Reply:
I pushed your suggestions, I can also do the same for `ask_and_set_test_framework`

Reply:
Great, thanks for your patience!